Makefile

```diff
@@ -0,0 +1,43 @@
+SWAGGER_CODEGEN_VERSION=2.2.3
+SWAGGER_CODEGEN_JAR_URL=http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.2.3/swagger-codegen-cli-$(SWAGGER_CODEGEN_VERSION).jar
```

Comment:
@pierre I think we can make the url completely dynamic to: `http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/$(SWAGGER_CODEGEN_VERSION)/swagger-codegen-cli-$(SWAGGER_CODEGEN_VERSION).jar` because if version will change, version folder itself will change.
Hope my syntax is correct! :)

Reply:
done